ynput/OpenPype
# Resolve: inventory management fix

## Changelog Description
Loaded Timeline item containers are now updating correctly and version management is working as it suppose to.

- [x] updating loaded timeline items
- [x] Removing of loaded timeline items

> Warning! For versions under 18.5 Beta 3
> The removal of loaded containers has reached the limits of Resolve API (v18.0.4). We can only remove the linked mediapool item, not the timeline item. The removed containers will leave the offline timeline item intact, requiring manual removal.

## Testing notes:
1. Load multiple products from couple different folder contexts. Make sure multiple versions of each are available and load some older versions.
2. Open `Manager ...` from OpenPype menu and update all versions to latest
![image](https://github.com/ynput/OpenPype/assets/40640033/3848cf24-9a00-4d26-896f-3a70a9dc79af)
